nemo_skills/evaluation/metrics/answer_judgement_metrics.py

```diff
@@ -57,16 +89,22 @@ def _update_score_metrics_for_pass(
     ):
         assert score_method == 'correct_judgements'
         # expected answer is always the same for all predictions, so just take the first one
-        gt_judgement = is_correct_judgement(predictions[0]['expected_judgement'])
-        pred_judgement = is_correct_judgement(predictions[0]['judgement'])
-        # if pass is not correct, means all predictions are the same and wrong
-        if not pass_score:
-            self._update_fp_fn(eval_dict[f"pass@{k}"], pred_judgement, gt_judgement)
+        gt_judgement = is_correct_judgement_or_none(predictions[0]['expected_judgement'])
+        pred_judgements = [is_correct_judgement_or_none(pred['judgement']) for pred in predictions]
```

Comment:
```suggestion
        pred_judgements = [is_correct_judgement_or_none(pred['judgement']) for pred in predictions[:k]]
```
Right now we pass full list of predictions (in case you want to do some bootstrapping or things like this to calculate @ k metrics), so need to pick the right subset